packages/shorebird_cli/lib/src/command_runner.dart

```diff
@@ -181,7 +181,7 @@ ${lightCyan.wrap('shorebird release android -- --no-pub lib/main.dart')}''';
     }
 
     // Run the command or show version
-    final int? exitCode;
+    int? exitCode;
```

Comment:
why did we have to make this non-final?

Reply:
Mostly because the analyzer isn't smart enough to detect that `exitCode = await super.runCommand(topLevelResults);` is the only statement in the `try` block. The analyzer thinks there's a chance that it will get assigned a value and that the `catch` block (where it's assigned a value) may also execute.